bridgestyle/sld/fromgeostyler.py

```diff
@@ -8,6 +8,35 @@
 
 _warnings = []
 
+# return a dictionary<int,list of rules>, where int is the Z value
+# symbolizers are marked with a Z
+#
+# a rule (with multiple sybolizers) will have the rule replicated, one for each Z value found in the symbolizer
+#
+# ie. rule[0]["symbolizers"][0] has Z=0
+#     rule[0]["symbolizers"][1] has Z=1
+#
+# this will return
+#   result[0] => rule with symbolizer 0  (name changed to include Z=0)
+#   result[1] => rule with symbolizer 1  (name changed to include Z=1)
+def processRulesByZ(rules):
+    result = {}
+    for rule in rules:
+        for symbolizer in rule["symbolizers"]:
+            z = 0
```

Comment:
these lines can be replaced with `z = symbolizer.get("Z", 0)`

Reply:
updated